src/aspire/aspire/utils/compare_stacks.py

```diff
@@ -62,9 +59,6 @@ def compare_stacks(stack1, stack2, verbose=None, max_error=None):
         if verbose == 0:
             continue
 
-        elif verbose == 1:
```

Comment:
While we're here, can you remove the blank line above so that the if and elif blocks stick together?

Reply:
sure, I rm'd the blanks about bot elif blocks.